src/jdk.jdwp.agent/share/native/libjdwp/log_messages.c

```diff
@@ -78,9 +78,12 @@ get_time_stamp(char *tbuf, size_t ltbuf)
                 "%d.%m.%Y %T", localtime(&t));
     (void)strftime(timestamp_timezone, TZ_SIZE,
                 "%Z", localtime(&t));
-    (void)snprintf(tbuf, ltbuf,
-                   "%s.%.3d %s", timestamp_date_time,
-                   (int)(millisecs), timestamp_timezone);
+
+    // Truncate milliseconds in buffer large enough to hold the
+    // value which is always < 1000 (and so a maximum of 3 digits for "%.3s")
+    char tmp[10 + 1];
```

Comment:
I was wrong yesterday. Max len of %n would be 11 chars (if INT_MIN). Can you make this buffer 11 chars please? 

This error would have had no practical consequence: tmp[] would be filled to the brim, leaving out the terminating zero, and since we then print with %.3s, this would have had no negative effect)

Reply:
> INT_MIN = "-2147483647" = 11
If millisecs is unsigned do we not  have to account for the minus sign?

Why can't tmp just be 20 and we don't have to count characters since snprintf will null terminate it?

Reply:
Sure, thats fine too!